### PR TITLE
Fix: Cledwin missing feature

### LINKED ISSUE #147: Cledwyn npc needs to enchant amulets
cledwyn

These are the amulets Cledwyn should enchant for 2 silver tokens.

### LINKED ISSUE #1588: Cledwyn npc needs to enchant amulets
cledwyn

These are the amulets Cledwyn should enchant for 2 silver tokens.

### PR DESCRIPTION
**Pull request by marcosvf132**
Resolves #1588
- **Add** 'enchant' option;
- **Fix** items 'pendulet', 'amulet of theurgic' and 'sleep shawl';
- **Add** 'addon' option: first and second rift warrior addon + achievement;
- **Fix** conflict on 'Faceless bane' and 'The nightmare beast' with the `items.xml` commit changes. (Non unique item name).

### CODE REVIEW

File: data/items/items.xml [hunk 1]
```diff
@@ -45336,23 +45336,32 @@
 		<attribute key="duration" value="600" />
 		<attribute key="showduration" value="1" />
 	</item>
-	<item id="34980" article="an" name="enchanted sleep shawl">
+	<item id="34980" article="a" name="enchanted sleep shawl">
```
Comment (attack787) [inline data/items/items.xml]: is "an" and not "a"

File: data/items/items.xml [hunk 2]
```diff
@@ -45336,23 +45336,32 @@
...
-		<attribute key="decayTo" value="34981" />
+		<attribute key="transformDeEquipTo" value="34981" />
+		<attribute key="decayTo" value="34066" />
 		<attribute key="duration" value="3600" />
 		<attribute key="showduration" value="1" />
 		<attribute key="showattributes" value="1" />
 	</item>
-	<item id="34981" article="a" name="sleep shawl">
-		<attribute key="weight" value="650" />
+	<item id="34981" article="a" name="enchanted sleep shawl">
```
Comment (attack787) [inline data/items/items.xml:45353]: here too, "an"

File: data/items/items.xml [hunk 3]
```diff
@@ -45362,15 +45371,23 @@
...
 		<attribute key="absorbPercentPhysical" value="5" />
-		<attribute key="decayTo" value="34983" />
+		<attribute key="transformDeEquipTo" value="34983" />
+		<attribute key="decayTo" value="34067" />
 		<attribute key="duration" value="7200" />
 		<attribute key="showduration" value="1" />
 		<attribute key="showattributes" value="1" />
 	</item>
-	<item id="34983" article="a" name="pendulet">
+	<item id="34983" article="a" name="enchanted pendulet">
```
Comment (attack787) [inline data/items/items.xml:45380]: and here

File: data/items/items.xml [hunk 4]
```diff
@@ -45336,23 +45336,32 @@
...
 		<attribute key="duration" value="3600" />
 		<attribute key="showduration" value="1" />
 		<attribute key="showattributes" value="1" />
 	</item>
-	<item id="34981" article="a" name="sleep shawl">
-		<attribute key="weight" value="650" />
+	<item id="34981" article="a" name="enchanted sleep shawl">
+		<attribute key="weight" value="430" />
 		<attribute key="slotType" value="necklace" />
 		<attribute key="quicklootcategory" value="amulet" />
```
Comment (attack787) [inline data/items/items.xml:45356]: I think don't need more quicklootcategory

**Comment by marcosvf132** [comment]
Ops I didn't notice these article change, I will upload with the right change.
The **quicklootcategory** I won't change anything, at least not now because a lot of items on `items.xml` still have this option and I don't know if this option is deprecated or not.

File: data/items/items.xml (hunk 4 above)
Comment (majestyotbr) [reply data/items/items.xml:45356]: The quicklootcategory is not deprecated yet.

File: data/items/items.xml [hunk 5]
```diff
@@ -44549,7 +44549,7 @@
 		<attribute key="absorbPercentIce" value="3" />
 		<attribute key="imbuingSlots" value="2" />
 	</item>
-	<item id="34066" article="a" name="sleep shawl">
+	<item id="34066" article="an" name="sleep shawl">
```
Comment (majestyotbr) [inline data/items/items.xml]: `<item id="34066" article="a" name="sleep shawl">`

File: data/items/items.xml [hunk 6]
```diff
@@ -50620,7 +50620,7 @@
 		<attribute key="slotType" value="necklace" />
 		<attribute key="weight" value="430" />
 	</item>
-	<item id="34067" article="a" name="pendulet">
+	<item id="34067" article="an" name="pendulet">
```
Comment (majestyotbr) [inline data/items/items.xml]: `<item id="34067" article="a" name="pendulet">`

Comment (majestyotbr) [reply data/items/items.xml]: Put the id 34067 in order, follow the correct sequence.

**Review by majestyotbr — APPROVED** [review]

**Pull request merged**